test.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+require './capitalize_Decorator'
+require './trimmer_decorator'
+
+person = Person.new(22, 'maximilianus')
+capitalize_person = CapitalizeDecorator.new(person)
+trim_person = TrimmerDecorator.new(person)
```

Comment:
- [ ] **[OPTIONAL]** Just for reference you should have passed the capitalize_person, instead of person to the trimmer

  ```
  trim_person = TrimmerDecorator.new(capitalize_person)
  ```